parcel/parcel.jl

```diff
@@ -99,6 +99,15 @@ function parcel_model(dY, Y, p, t)
         end
         dqi_dt_new_depo = dN_act_dt_depo * 4 / 3 * FT(π) * r_nuc^3 * ρ_ice / ρ_air
     end
+    if "ActivityBasedDeposition" in ice_nucleation_modes
```

Comment:
I think we are assuming here that someone will either run with `MohlerAF` `MohlerRate` or `ActivityBased` because each time we are doing `dqi_dt_new_depo = ...` and `dN_act_dt_depo = ...` instead of `+=`. I think thats fine, but maybe we could add some error warning to highlight that.

I think because the aerosol types don't overlap between `Mohler` and `Activity` based, one could try to run both at the same time